tensorflow/training/docker/1.15/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,224 @@
+FROM nvidia/cuda:11.0-base-ubuntu18.04
```

Comment:
what's the difference between this file and the one in the 1.15.4 dir? can we remove the 1.15.4 dir altogether?

Reply:
Removed